code/api/src/seeders/1-user.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
         email: 'admin@crate.com',
         password: bcrypt.hashSync('123456', config.saltRounds),
         role: params.user.roles.admin,
+        style: 'Classy',
```

Comment:
Helpful to confirm that the `style` attribute has been added to the DB

Reply:
I think this answers my question about the difference...maybe?